src/main/java/com/monumental/models/Monument.java

```diff
@@ -249,7 +257,8 @@ private String generateDescription() {
                 Reference firstReference = this.references.get(0);
 
                 if (firstReference != null && firstReference.getUrl() != null) {
-                    description += " You may find further information about this monument at: " + firstReference.getUrl();
+                    description += " You may find further information about this monument or memorial at: " +
+                            firstReference.getUrl();
```

Comment:
Sort of related to the image gallery we may want a way to signify a primary reference rather than just take the oldest